src/main/java/net/devtech/arrp/json/blockstate/JVariant.java

```diff
@@ -13,50 +15,51 @@
 import net.minecraft.util.math.Direction;
 
 public final class JVariant implements Cloneable {
-	private final Map<String, JBlockModel> models = new HashMap<>();
+	private final Map<String, List<JBlockModel>> models = new HashMap<>();
```

Comment:
You may need to create a gson serializer like some of the other classes to handle the case where there is only one model. I don't think it's *wrong* per-say but I think it should be done

Reply:
Done!